tests/integration/dogshell/test_dogshell.py

```diff
@@ -134,115 +113,83 @@ def match_permalink(out):
-        out, err, return_code = self.dogshell(["tag", "show", host])
+        out, _, _ = self.dogshell(["tag", "replace", host] + tags1)
         for t in tags1:
-            assert t in out, (t, out)
+            assert t in out
         for t in tags0:
-            assert t not in out, (t, out)
+            assert t not in out
 
         # Remove all the tags
-        self.dogshell(["tag", "detach", host])
-        out, err, return_code = self.dogshell(["tag", "show", host])
-        self.assertEqual(out, "")
+        out, _, _ = self.dogshell(["tag", "detach", host])
+        assert out == ""
 
     def test_timeboards(self):
         # Create a timeboard and write it to a file
         name, temp0 = get_temp_file()
         graph = {
             "title": "test metric graph",
-            "definition":
-                {
-                    "requests": [{"q": "testing.metric.1{host:blah.host.1}"}],
-                    "viz": "timeseries",
-                }
+            "definition": {"requests": [{"q": "testing.metric.1{host:blah.host.1}"}], "viz": "timeseries"},
         }
         self.dogshell(["timeboard", "new_file", name, json.dumps(graph)])
         dash = json.load(temp0)
-        assert 'id' in dash, dash
-        assert 'title' in dash, dash
+        assert "id" in dash
+        assert "title" in dash
 
         # Update the file and push it to the server
         unique = self.get_unique()
-        dash['title'] = 'dash title %s' % unique
+        dash["title"] = "dash title {}".format(unique)
         name, temp1 = get_temp_file()
         json.dump(dash, temp1)
         temp1.flush()
         self.dogshell(["timeboard", "push", temp1.name])
 
         # Query the server to verify the change
-        out, _, _ = self.dogshell(["timeboard", "show", str(dash['id'])])
+        out, _, _ = self.dogshell(["timeboard", "show", str(dash["id"])])
 
         out = json.loads(out)
-        assert "dash" in out, out
-        assert "id" in out["dash"], out
-        self.assertEqual(out["dash"]["id"], dash["id"])
-        assert "title" in out["dash"]
-        self.assertEqual(out["dash"]["title"], dash["title"])
+        out["dash"]["id"] == dash["id"]
+        out["dash"]["title"] == dash["title"]
 
         new_title = "new_title"
         new_desc = "new_desc"
-        new_dash = [{
-                    "title": "blerg",
-                    "definition": {
-                        "viz": "timeseries",
-                        "requests": [
-                            {"q": "avg:system.load.15{web,env:prod}"}
-                        ]
-                    }
-                    }]
+        new_dash = [
+            {
+                "title": "blerg",
+                "definition": {"viz": "timeseries", "requests": [{"q": "avg:system.load.15{web,env:prod}"}]},
+            }
+        ]
 
         # Update a dash directly on the server
-        self.dogshell(["timeboard", "update", str(dash["id"]), new_title, new_desc],
```

Comment:
not updating anymore here?

Reply:
yes we are. It's the same command, just wrapped to respect the 120 chars per line